pages/index.js

```diff
@@ -16,26 +16,47 @@ import GitHubAccelerator from "@components/GitHubAccelerator";
 import Alert from "@components/Alert";
 import CallToAction from "@components/CallToAction";
 import UserMini from "@components/user/UserMini";
+import { serverEnv } from "@config/schemas/serverSchema";
 
 export async function getStaticProps() {
-  const pageConfig = config.isr.homepage; // Fetch the specific configuration for this page
+  const pageConfig = config.isr.homepage;
 
   const { stats: totalStats } = await getTotalStats();
   const { stats: todayStats } = await getTodayStats();
   const randomProfile = await getRandomProfileApi();
 
+  let alerts = config.alerts;
+  if (
+    process.env.NEXT_PUBLIC_VERCEL_ENV !== "production" &&
+    serverEnv.NODE_ENV === "development" &&
+    totalStats.users === 0
+  ) {
+    alerts.push({
+      type: "warning",
+      message:
+        'To (re)load json files and any changes, please visit "/api/system/reload?secret=development"',
+    });
+  }
+
   return {
     props: {
       BASE_URL: clientEnv.NEXT_PUBLIC_BASE_URL,
       total: totalStats,
       today: todayStats,
       randomProfile,
+      alerts: config.alerts,
```

Comment:
this should use the alerts variable the new alert was pushed to not the one from config,